packages/language-server/src/completion/completions.json

```diff
@@ -31,6 +31,17 @@
     {
       "label": "Decimal",
       "documentation": "Decimal value"
+    },
+    {
+      "label": "Unsupported(\"\")",
+      "documentation": "An arbitrary, unsupported database type",
+      "fullSignature": "Unsupported(_ name: String)",
+      "params": [
+        {
+          "label": "name",
+          "documentation": "Name of the database type"
```

Comment:
@do4gr this too

Reply:
Name of the database type. This will be passed on by Prisma Migrate to the underlying database  when creating migrations. Example: `myField Unsupported("polygon")?` will render 

```
CREATE TABLE ...
    ...
    `myField` polygon,
```

Reply:
So it needs to be the valid raw sql needed to create the type in the database. We will interpolate it unchecked into the DDL. Migrate will run no validations on it. It will not be present in your client API since it is not supported there.

Thats the necessary content, phrasing needs some work.
An example is a good idea, I agree. Best would be one with string quoting. 

myField Unsupported("''::citext")

```sql 
CREATE TABLE ...
    ...
    `myField` ''::citext,

```